src/components/Analytics/detailedCharts/answersDistribution/AnswersDistributionMultipleChoices.js

```diff
@@ -0,0 +1,96 @@
+import { useMemo } from 'react';
+
+import { useTheme } from '@mui/material';
+
+import { truncateText } from '../../../../utils/plotUtils';
+import AnswersDistributionBarChart from './AnswersDistributionBarChart';
+
+/**
+ * Component used to display the answers distribution for the multiple choices question type
+ *
+ * @param maxWidth The max width of the chart
+ * @param question The question for which to display detailed information into the chart
+ * @param appDataForQuestion The app data for the question (i.e. users answers)
+ */
+const AnswersDistributionMultipleChoices = ({
+  maxWidth,
+  question,
+  appDataForQuestion,
+}) => {
+  const theme = useTheme();
+  const correctChoices = useMemo(
+    () =>
+      question.data.choices
+        .filter(({ isCorrect }) => isCorrect)
+        .map(({ value }) => value),
+    [question]
+  );
+
+  const questions = useMemo(
+    () =>
+      question.data.choices.reduce((acc, c) => {
+        acc[c.value] = 0;
+        return acc;
+      }, {}),
+    [question]
+  );
+
+  const responsesCount = useMemo(
+    () =>
+      appDataForQuestion.reduce(
+        (questionsCountAcc, appData) =>
+          appData.data.choices.reduce((acc, choice) => {
+            acc[choice]++;
+            return acc;
+          }, questionsCountAcc),
+        questions
+      ),
+    [appDataForQuestion, questions]
+  );
```

Comment:
Either add a comment to say that the first reduce is an initializer array, or combine both (I guess it's a matter of adding a if clause)

Reply:
I added a comment on top of the first reduce to tell that it was an initializer.

I prefer to do it like that, as after I reduce the `appData`, which might not contains all the choices, and I'd like to display if a choice was selected 0 times. So I initialize all the choices to 0, using all the possible choices from the question directly.